### PR TITLE
Update PHPStan level to 4

### PR DESCRIPTION
**Pull request by westonruter**
See #775.

This fixes the following PHPStan issues:

<details><summary>[ERROR] Found 65 errors</summary>

```
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/dominant-color-images/class-dominant-color-image-editor-gd.php                                                                           
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  30     Negated boolean expression is always false.                                                                                                      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  60     Negated boolean expression is always false.                                                                                                      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/dominant-color-images/class-dominant-color-image-editor-imagick.php                                                                      
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  30     Negated boolean expression is always false.                                                                                                      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  62     Negated boolean expression is always false.                                                                                                      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  73     If condition is always true.                                                                                                                     
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------ 
  Line   plugins/dominant-color-images/helper.php                                                                                                              
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------ 
  136    Function dominant_color_get_dominant_color() never returns string so it can be removed from the return type.                                          
  145    Offset 'dominant_color' on array{width: int, height: int, file: string, sizes: array, image_meta: array, filesize: int} in isset() does not exist.    
  149    Unreachable statement - code above always terminates.                                                                                                 
  160    Function dominant_color_has_transparency() never returns bool so it can be removed from the return type.                                              
  166    Offset 'has_transparency' on array{width: int, height: int, file: string, sizes: array, image_meta: array, filesize: int} in isset() does not exist.  
  170    Unreachable statement - code above always terminates.                                                                                                 
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------ 

 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/webp-uploads/helper.php                                                                                                                                        
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  51     Call to function is_array() with array will always evaluate to true.                                                                                                   
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                        
  145    Offset 'height' on array{width: int, height: int, crop: bool} in isset() always exists and is not nullable.                                                            
  146    Offset 'width' on array{width: int, height: int, crop: bool} in isset() always exists and is not nullable.                                                             
  147    Offset 'crop' on array{width: int, height: int, crop: bool} in isset() always exists and is not nullable.                                                              
  182    Offset 'path' on array{path: string, file: non-falsy-string, width: int, height: int, mime-type: string, filesize: int} in isset() always exists and is not nullable.  
  208    Negated boolean expression is always false.                                                                                                                            
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                        
  288    Call to function is_array() with array will always evaluate to true.                                                                                                   
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                        
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/webp-uploads/hooks.php                                                                                                                                                                                                       
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  266    Offset 'function' on array{function: string, line?: int, file?: string, class?: class-string, type?: '->'|'::', args?: array, object?: object} in isset() always exists and is not nullable.                                         
  336    Negated boolean expression is always false.                                                                                                                                                                                          
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                      
  628    Offset 'sizes' on array{width: int, height: int, file: non-falsy-string, sizes: array, image_meta: array, filesize: int, sources?: array<string, array{file: string, filesize: int}>} in isset() always exists and is not nullable.  
  628    Right side of && is always true.                                                                                                                                                                                                     
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                      
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/webp-uploads/image-edit.php                                                                                                                                                           
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  253    Offset 'function' on array{function: string, line?: int, file?: string, class?: class-string, type?: '->'|'::', args?: array, object?: object} in isset() always exists and is not nullable.  
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/includes/admin/load-tests.php                                                                                                              
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  28     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'options-general.php' and array{} will always evaluate to false.               
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/includes/admin/server-timing-tests.php                                                                                                     
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  49     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'tools.php' and array{} will always evaluate to false.                         
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  61     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'perflab-server…' and array{} will always evaluate to false.                   
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  70     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'perflab-server…' and array{} will always evaluate to false.                   
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ -------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/includes/server-timing/load-tests.php                                                                                                       
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------- 
  15     Call to method PHPUnit\Framework\Assert::assertInstanceOf() with 'Perflab_Server_Timing' and Perflab_Server_Timing will always evaluate to true.  
  81     Call to method PHPUnit\Framework\Assert::assertTrue() with false will always evaluate to false.                                                   
  81     Offset 'perflab-server…' on array{} in isset() does not exist.                                                                                    
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/includes/server-timing/perflab-server-timing-tests.php                                                             
 ------ ------------------------------------------------------------------------------------------------------------------------- 
  73     Call to method PHPUnit\Framework\Assert::assertTrue() with false and 'Measure callback…' will always evaluate to false.  
 ------ ------------------------------------------------------------------------------------------------------------------------- 

 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/includes/site-health/audit-enqueued-assets/audit-enqueued-assets-test.php                                                                               
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  167    Call to method PHPUnit\Framework\Assert::assertIsArray() with array{direct: array<string, array{label: string, test: string}>} will always evaluate to true.  
 ------ -------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/plugins/optimization-detective/storage/data-tests.php                                                                                      
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  533    Call to method PHPUnit\Framework\Assert::assertTrue() with true will always evaluate to true.                                                    
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  533    Result of || is always true.                                                                                                                     
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  533    Strict comparison using === between false and false will always evaluate to true.                                                                
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  535    Call to method PHPUnit\Framework\Assert::assertIsString() with string will always evaluate to true.                                              
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/plugins/speculation-rules/speculation-rules-helper-test.php                                                                                
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  37     Call to method PHPUnit\Framework\Assert::assertIsArray() with array will always evaluate to true.                                                
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/plugins/webp-uploads/helper-tests.php                                                                                                        
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------- 
  101    Call to method PHPUnit\Framework\Assert::assertIsArray() with array{file: string, filesize: int} will always evaluate to true.                     
  102    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'filesize' and array{file: string, filesize: int} will always evaluate to true.  
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  103    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'file' and array{file: string, filesize: int} will always evaluate to true.      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  128    Call to method PHPUnit\Framework\Assert::assertIsArray() with array{file: string, filesize: int} will always evaluate to true.                     
  129    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'filesize' and array{file: string, filesize: int} will always evaluate to true.  
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  130    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'file' and array{file: string, filesize: int} will always evaluate to true.      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  292    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'filesize' and array{file: string, filesize: int} will always evaluate to true.  
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  293    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'file' and array{file: string, filesize: int} will always evaluate to true.      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  326    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'filesize' and array{file: string, filesize: int} will always evaluate to true.  
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  328    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'file' and array{file: string, filesize: int} will always evaluate to true.      
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  370    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array<string>> will always evaluate to true.                           
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  390    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array<string>> will always evaluate to true.                           
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  409    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array<string>> will always evaluate to true.                           
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  428    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array<string>> will always evaluate to true.                           
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
  444    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array<string>> will always evaluate to true.                           
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.    
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/plugins/webp-uploads/image-edit-tests.php                                                                                                                                                                                                                             
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  40     Call to method PHPUnit\Framework\Assert::assertIsArray() with Countable will always evaluate to false.                                                                                                                                                                      
  44     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'full-orig' and array{} will always evaluate to false.                                                                                                                                                    
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  54     Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'sources' and *NEVER* will always evaluate to true.                                                                                                                                                       
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  277    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'original_image' and array{width: int, height: int, file: string, sizes: array, image_meta: array, filesize: int, sources?: array<string, array{file: string, filesize: int}>}|null will always evaluate  
         to false.                                                                                                                                                                                                                                                                   
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  278    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'sources' and *NEVER* will always evaluate to true.                                                                                                                                                       
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  287    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'sizes' and *NEVER* will always evaluate to true.                                                                                                                                                         
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  327    Call to method PHPUnit\Framework\Assert::assertArrayHasKey() with 'sizes' and array{width: int, height: int, file: string, sizes: array, image_meta: array, filesize: int, sources: array<string, array{file: string, filesize: int}>} will always evaluate to true.        
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.                                                                                                                             
  493    Call to method PHPUnit\Framework\Assert::assertSame() with 'full-orig' and 'full-orig' will always evaluate to true.                                                                                                                                                        
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/plugins/webp-uploads/load-tests.php                                                                                                        
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  250    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array{file: string, filesize: int}> will always evaluate to true.    
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  705    Call to method PHPUnit\Framework\Assert::assertIsArray() with array<string, array{file: string, filesize: int}> will always evaluate to true.    
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/utils/Constraint/ImageHasSource.php                                                                                                        
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 
  72     Offset 'sources' on array{width: int, height: int, file: string, sizes: array, image_meta: array, filesize: int} in isset() does not exist.      
  72     Result of || is always true.                                                                                                                     
         💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.  
  78     Unreachable statement - code above always terminates.                                                                                            
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------- 

```

</details> 

Please review individual commits for changes.

### CODE REVIEW

**Comment by westonruter** [comment]
Remaining issues to resolve:

```
 ------ ----------------------------------------------------------------------- 
  Line   tests/includes/server-timing/load-tests.php                            
 ------ ----------------------------------------------------------------------- 
  15     Call to method PHPUnit\Framework\Assert::assertInstanceOf() with       
         'Perflab_Server_Timing' and Perflab_Server_Timing will always          
         evaluate to true.                                                      
  81     Call to method PHPUnit\Framework\Assert::assertTrue() with false will  
         always evaluate to false.                                              
  81     Offset 'perflab-server…' on array{} in isset() does not exist.         
 ------ ----------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   tests/includes/server-timing/perflab-server-timing-tests.php          
 ------ ---------------------------------------------------------------------- 
  73     Call to method PHPUnit\Framework\Assert::assertTrue() with false and  
         'Measure callback…' will always evaluate to false.                    
 ------ ---------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------- 
  Line   tests/includes/site-health/audit-enqueued-assets/audit-enqueued-assets-test.php  
 ------ --------------------------------------------------------------------------------- 
  167    Call to method PHPUnit\Framework\Assert::assertIsArray() with array              
         will always evaluate to true.                                                    
 ------ --------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------ 
  Line   tests/plugins/webp-uploads/helper-tests.php                       
 ------ ------------------------------------------------------------------ 
  101    Call to method PHPUnit\Framework\Assert::assertIsArray() with     
         array{file: string, filesize: int} will always evaluate to true.  
  128    Call to method PHPUnit\Framework\Assert::assertIsArray() with     
         array{file: string, filesize: int} will always evaluate to true.  
 ------ ------------------------------------------------------------------ 

 ------ --------------------------------------------------------------- 
  Line   tests/plugins/webp-uploads/image-edit-tests.php                
 ------ --------------------------------------------------------------- 
  40     Call to method PHPUnit\Framework\Assert::assertIsArray() with  
         Countable will always evaluate to false.                       
  493    Call to method PHPUnit\Framework\Assert::assertSame() with     
         'full-orig' and 'full-orig' will always evaluate to true.      
 ------ --------------------------------------------------------------- 

 [ERROR] Found 9 errors                                                                  
```

File: tests/includes/server-timing/perflab-server-timing-tests.php [hunk 1]
```diff
@@ -53,21 +53,31 @@ public function test_register_metric_stores_metrics_and_runs_measure_callback()
...
+			)
 		);
 
-		$this->server_timing->register_metric( 'test-metric', $args );
-
 		$this->assertTrue( $this->server_timing->has_registered_metric( 'test-metric' ), 'Metric without cap should still be registered' );
 		$this->assertFalse( $called, 'Measure callback without cap must not be run' );
 
 		wp_set_current_user( self::$admin_id );
-		$this->server_timing->register_metric( 'test-metric-2', $args );
```
Comment (westonruter) [inline tests/includes/server-timing/perflab-server-timing-tests.php:70]: Without passing the callback here, PHPStan isn't aware that the `$called` variable could actually have changed so it complains:

> Call to method PHPUnit\Framework\Assert::assertTrue() with false and 'Measure callback…' will always evaluate to false.

File: tests/plugins/webp-uploads/image-edit-tests.php [hunk 2]
```diff
@@ -36,8 +36,12 @@ public function it_should_backup_the_sources_structure_alongside_the_full_size()
 
 		$backup_sizes = get_post_meta( $attachment_id, '_wp_attachment_backup_sizes', true );
 
-		$this->assertNotEmpty( $backup_sizes );
 		$this->assertIsArray( $backup_sizes );
+		$this->assertArrayHasKey( 'full-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'thumbnail-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'medium-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'medium_large-orig', $backup_sizes );
+		$this->assertArrayHasKey( 'large-orig', $backup_sizes );
```
Comment (westonruter) [inline tests/plugins/webp-uploads/image-edit-tests.php:44]: Without assertions like this, PHPStan complains about the `foreach()` loop below being passed `$backup_sizes` as an empty array.

File: tests/plugins/webp-uploads/image-edit-tests.php [hunk 3]
```diff
@@ -489,9 +493,7 @@ public function it_should_store_the_next_image_hash_on_the_backup_sources() {
 		$this->assertIsArray( $backup_sources );
 		$this->assertCount( 2, $backup_sources );
 		foreach ( array_keys( $backup_sources ) as $name ) {
-			if ( 'full-orig' === $name ) {
-				$this->assertSame( 'full-orig', $name );
```
Comment (westonruter) [inline tests/plugins/webp-uploads/image-edit-tests.php:493]: Clearly this will always be the same 😄

File: includes/server-timing/load.php [hunk 4]
```diff
@@ -51,8 +51,16 @@ function perflab_server_timing(): Perflab_Server_Timing {
...
+/**
+ * Initializes the Server-Timing API.
+ *
+ * @since n.e.x.t
+ */
+function perflab_server_timing_init(): void {
+	perflab_server_timing();
+}
+
+add_action( 'wp_loaded', 'perflab_server_timing_init' );
```
Comment (westonruter) [inline includes/server-timing/load.php:63]: This just avoids the need to ignore the PHPStan error about an action callback returning a value.

**Review by mukeshpanchal27 — APPROVED** [review]

**Pull request merged**